src/cmap/conn/mod.rs

```diff
@@ -323,6 +348,7 @@ impl From<DroppedConnectionState> for Connection {
             stream_description: state.stream_description.take(),
             ready_and_available_time: None,
             pool: None,
+            partial_message_state: state.partial_message_state.clone(),
```

Comment:
I think we can just move the partial message state here instead of cloning

Reply:
Ah makes sense. I think clippy would've caught it anyway if this clone was unnecessary.